### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by ChristophLeuker**
add a navbar and a heading to the landing page. The navbar will come from the left if you push that Button above.

### CODE REVIEW

File: components/landingNavbar/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+const NavbarLink = styled(Link)`
+  color: #333;
+  text-decoration: none;
+  margin-bottom: 0.5rem;
+  border: solid black 1px;
+  border-radius: 100px;
+  padding: 10px;
+  background-color: #ff9900;
+
+  &:hover {
```
Comment (ryliecc) [inline components/landingNavbar/index.js:31]: do we use hover when designing for mobile apps? I think you'll be fine leaving it in, but you don't need it here. A finger touching a screen can not "hover" 🙃

File: components/landingNavbar/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  display: inline-block;
+  background-color: orange;
+  border: solid black 1px;
+  color: #333;
+  font-size: 1.2rem;
+  cursor: pointer;
+  margin-left: auto;
+  border-radius: 50px;
+
+  &:hover {
```
Comment (ryliecc) [inline components/landingNavbar/index.js:50]: same here.

**Review by ryliecc — APPROVED** [review]
Well written, easy to understand. LGTM ⭐️

File: components/landingNavbar/index.js (hunk 1 above)
Comment (ChristophLeuker) [reply components/landingNavbar/index.js:31]: True, but i think it can stay for the pc users ;)

**Pull request merged**